addons/mail/static/src/new/discuss/discuss.scss

```diff
@@ -7,3 +7,11 @@
         background-color: $gray-400;
     }
 }
+
+.o-mail-discuss-thread {
+    min-height: 0;
+}
```

Comment:
Prefer setting `overflow-auto` classname.

Flexbox states that flex item cannot shrink below their content by default.
It's like they have `min-width: auto` and `overflow: visible` of some sort.
The specs tells that `min-width: auto` follows `overflow` rule, which is `visible` here.

Telling `min-width: 0` by-passes `overflow` rule, whereas `overflow-auto` makes sure both are coherent and live together in harmony.

A more pragmatic motivation is that `overflow-auto` is shorter than defining this css rule. There's no classname with `min-width`, which is actually convenient in this case.